server/game/cards/06-CotE/OutskirtsSentry.js

```diff
@@ -0,0 +1,23 @@
+const DrawCard = require('../../drawcard.js');
+const { CardTypes } = require('../../Constants');
+
+class OutskirtsSentry extends DrawCard {
+    setupCardAbilities(ability) {
+        this.reaction({
+            title: 'Honor a participating character',
+            when: {
+                onMoveToConflict: (event, context) => context.source.isParticipating()
+            },
+            collectiveTrigger: true,
```

Comment:
This ability shouldnt have collectiveTrigger because it doesn't say 'After one or more characters move to a conflict....'